commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/Eviction.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.commons.util.concurrent.jdk8backported;
+
+/**
+ * @author Will Burns
```

Comment:
So maybe the javadocs should be simply removed... an empty thing is just rubbish...